src/user/user.controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Delete,
+  Param,
+  ParseIntPipe,
+} from '@nestjs/common';
+import { CreateUserDto } from './dtos/create-user.dto';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+  constructor(private userService: UserService) {}
+  @Get('/all')
+  async getAllUsers() {
+    const allUsers = await this.userService.findAll();
+    return allUsers;
+  }
+
+  @Get(':id')
+  async getUserById(@Param('id', ParseIntPipe) id: number) {
+    const user = await this.userService.findOne(id);
+    return user;
+  }
+
+  @Post('/add')
```

Comment:
```suggestion
  @Post()
```

Không đúng chuẩn Rest API. Tìm hiểu lại